.drone.yml

```diff
@@ -151,14 +150,12 @@ services:
       event: [push, pull_request]
 
   selenium:
-    image: selenium/standalone-chrome-debug:3.10.0
+    image: selenium/standalone-chrome:3.10.0
```

Comment:
We should use debug here, see related comment
https://github.com/fidals/stroyprombeton/pull/291#discussion_r209157758